src/components/record/RecordsController.jsx

```diff
@@ -104,6 +104,7 @@ class RecordsController extends React.Component {
       this.props.exportRecords(exportType, {
         ...this.state.filters,
         sort: sortToParams(this.state.sort),
+        page: this.state.pageNumber,
```

Comment:
@kostobog I believe this is what this parameter is supposed to represent?

Reply:
@LaChope 
Yes I think this the right parameter. You should also set the size parameter, see how it is done in `_loadRecords()`.